### DISCUSSION TITLE
Does it work on flutter web?

### DISCUSSION THREAD

**Discussion by alexpavelm** [fzyzcjy/flutter_smooth#162] [General]
Will this package solve jank on flutter web on lower end mobile devices with canvaskit renderer? I can't test it myself because I am a complete newbie and I don't know how to make it work at the moment.

**Comment by fzyzcjy** [comment]
https://discord.com/channels/608014603317936148/608022171637252115/1032175080014364744

**Reply by alexpavelm** [reply]
Could you send me an invite link to the discord server? I can't access the link you've sent. Thanks!

**Reply by fzyzcjy** [reply]
flutter.dev has public invitations

**Reply by alexpavelm** [reply]
Well, I am already on the official flutter server, if that's the one you are talking about, but I just can't access the link you provided above. 
![image](https://user-images.githubusercontent.com/44024464/198983523-7b48b671-7b8a-4fd9-9d4b-906a48d588cf.png)

**Reply by fzyzcjy** [reply]
![image](https://user-images.githubusercontent.com/5236035/198989515-ad89b9a9-756e-4267-b432-04acea7e6aee.png)

click button in welcome channel
